src/lib/drupal/query.ts

```diff
@@ -135,6 +135,6 @@ export const entityBaseFields = (entity: NodeTypes): PublishedEntity => {
     title: entity.title,
     metatags: entity.metatag,
     breadcrumbs: entity.breadcrumbs,
-    lastUpdated: entity.field_last_saved_by_an_editor || entity.created,
+    lastUpdated: entity.field_last_saved_by_an_editor || entity.changed || entity.created,
```

Comment:
Slack thread for context: https://dsva.slack.com/archives/C07B86KV4P4/p1729097328269789?thread_ts=1729096713.224239&cid=C07B86KV4P4

content-build was handling this value a little differently; next-build simplifies the implementation of the "Last updated" timestamp but still needs to consider all 3 of these fields.